### PR TITLE
Update readme

### LINKED ISSUE #37: updating the ReadMe.md file
Updating the readme file to have the project information

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
I updated the readme file to have the project's info

### CODE REVIEW

**Comment by Moamin-AbuEwaida** [comment]
This PR includes Hasan's CSS files :thinking:

File: README.md [hunk 1]
```diff
@@ -1,9 +1,99 @@
...
+  - [Technologies](#technologies)
+  - [Code Examples](#code-examples)
+  - [Features](#features)
+  - [Status](#status)
+  - [Inspiration](#inspiration)
+  - [Contact](#contact)
+
+## General info
+
+A todo list , the objective is build the app using Javascript to create DOM elements during the user interaction with the app interface .
```
Comment (rayanejsilva) [inline README.md]: This group project is part of [HackYourFuture Belgium](https://hackyourfuture.be/) module **Separation of concerns**. Our objective is to build an app using Javascript to create DOM elements during the user interaction with the app interface .

File: README.md [hunk 2]
```diff
@@ -1,9 +1,99 @@
...
+## Screenshots
+
+![Example screenshot](client/public/ToDo-list-screen.png)
+
+## Technologies
+
+- JavaScript
+- HTML
+- CSS
+- VSC code
```
Comment (rayanejsilva) [inline README.md]: VSCode

**Review by rayanejsilva — CHANGES_REQUESTED** [review]
@Moamin-AbuEwaida Good job. I just added two small suggestions. :)

**Review by emelysalmeron — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Moamin-AbuEwaida) [reply README.md]: Oh yeah, would be great to add VS code :+1:

Comment (rayanejsilva) [reply README.md]: @Moamin-AbuEwaida you already added it before. My comment was about the spelling mistake. **VSC code** --> _VSCode_

**Review by rayanejsilva — APPROVED** [review]

**Pull request merged**